### ISSUE TITLE
[BUG] chars should not be allowed in ID, TYPE, ATTR_NAME, ATTR_VALUE fields

### ISSUE THREAD

**Issue by iariasleon** [telefonicaid/fiware-orion#1351]
#### Create request:
```
POST http://<host>:<port>/v2/entities
Fiware-Service: happy_path
Fiware-ServicePath: /test
Content-Type: application/json
{"id": "room_?", "type": "house", "timestamp": "34"}
```

#### Dataset:
```
  ?
  &
  /
  #
whitespace
```

#### Mongo:
```
{ "_id" : { "id" : "room_?", "type" : "house_?", "servicePath" : "/test" }, "attrNames" : [ "timestamp_?" ], "attrs" : { "timestamp_?" : { "type" : "", "creDate" : 1444724747, "modDate" : 1444724747, "value" : "34_?" } }, "creDate" : 1444724747, "modDate" : 1444724747 }
{ "_id" : { "id" : "room_&", "type" : "house_&", "servicePath" : "/test" }, "attrNames" : [ "timestamp_&" ], "attrs" : { "timestamp_&" : { "type" : "", "creDate" : 1444724775, "modDate" : 1444724775, "value" : "34_&" } }, "creDate" : 1444724775, "modDate" : 1444724775 }
{ "_id" : { "id" : "room_/", "type" : "house_/", "servicePath" : "/test" }, "attrNames" : [ "timestamp_/" ], "attrs" : { "timestamp_/" : { "type" : "", "creDate" : 1444724792, "modDate" : 1444724792, "value" : "34_/" } }, "creDate" : 1444724792, "modDate" : 1444724792 }
{ "_id" : { "id" : "room_#", "type" : "house_#", "servicePath" : "/test" }, "attrNames" : [ "timestamp_#" ], "attrs" : { "timestamp_#" : { "type" : "", "creDate" : 1444724809, "modDate" : 1444724809, "value" : "34_#" } }, "creDate" : 1444724809, "modDate" : 1444724809 }
```

#### Expected:
```
Status code: 400
{
     "error":"BadRequest",
     "description":"Invalid characters in ...."   (entity type, id, attribute name or attribute value)
}
 
```

**Comment by iariasleon** [comment]
LGTM
```
{"error":"BadRequest","description":"Invalid characters in entity type"}
{"error":"BadRequest","description":"Invalid characters in entity id"}
{"error":"BadRequest","description":"Invalid characters in attribute name"}
{"error":"BadRequest","description":"Invalid characters in attribute type"}
{"error":"BadRequest","description":"Invalid characters in metadata name"}
{"error":"BadRequest","description":"Invalid characters in metadata type"}
```

**Comment by fgalan** [comment]
(Chage milestone to 0.27.0. It seem this has ben wrongly closed in 0.29.0)
